packages/fx-core/resource/package.nls.json

```diff
@@ -1147,6 +1147,8 @@
   "driver.file.summary.withTarget": "The environment variables has been generated successfully to %s.",
   "driver.file.progressBar.title": "file/updateEnv",
   "driver.file.progressBar.generate": "Generating the environment variables to .env file.",
+  "driver.file.updateJson.description": "Add or update appsettings to JSON file.",
+  "driver.file.updateJson.summary.withTarget": "The appsettings has been generated successfully to %s.",
```

Comment:
suggest to be more specific:  The appsettings -> The value for appsettings

and it is aligned with singular form as well.

Reply:
Updated.